source/migration/BuildTasks/Xamarin.AndroidX.Migration.targets

```diff
@@ -204,7 +219,7 @@
     </Target>
 
     <Target Name="_AndroidXCecilfy"
-            DependsOnTargets="_AndroidXCecilfyShrinkFindFiles;_AndroidXCecilfyShrinkCopyFiles;_AndroidXCecilfyNoShrinkFindFiles"
+            DependsOnTargets="_ValidateLinkMode;_AndroidXCecilfyShrinkFindFiles;_AndroidXCecilfyShrinkCopyFiles;_AndroidXCecilfyNoShrinkFindFiles"
```

Comment:
This is required because there are 2 steps to running an app.

Build and Install.

Build depends on `_ValidateLinkMode` and that makes sure that the `AndroidLinkMode` is None when using the shared runtime.

Install does not and then uses SdkOnly. As a result, bad things happen:

 - Build uses `_ValidateLinkMode` to reset LinkMode to None
 - Install just uses SdkOnly and then it sees that the cecil output is not there
    - this triggers cecil to copy things
    - cecil says my stamp is done, so SKIP!
    - the install copies the non-cecil files from the copies, when in fact it should have been the andrid\assets because it was built as LinkMode=None